src/Infrastructure/Elastic/ElasticDocumentAdapter.php

```diff
@@ -22,7 +22,7 @@ public function bulk(BulkOperationInterface $command): callable|array
     {
         return $this->client->bulk([
             'body' => $command->build(),
-        ]);
+        ])->asArray();
```

Comment:
nit: since `asArray` tries to convert to array the function probably can't return `callable` anymore